src/pvecontrol/cluster.py

```diff
@@ -79,6 +97,9 @@ def is_healthy(self):
     def get_resources_nodes(self):
         return [resource for resource in self.resources if resource["type"] == "node"]
 
+    def get_node_ressources(self, node_name):
```

Comment:
I correct the typo "s".

I have the same trouble, I keep thinking we should pass parent to all resources instead of API.
But this is a major change I don't think we should treat this in this PR.